service/resource/deployment/master_deployment.go

```diff
@@ -22,8 +22,17 @@ func newMasterDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 	for i, masterNode := range customObject.Spec.Cluster.Masters {
 		capabilities := customObject.Spec.KVM.Masters[i]
 
+		storageType := key.StorageType(customObject)
+
+		// During migration, some TPOs do not have storage type set.
+		// This specifies a default, until all TPOs have the correct storage type set.
+		// tl;dr - this shouldn't be here. If all TPOs have storageType, remove it.
+		if storageType == "" {
```

Comment:
Please create an issue for this.

Reply:
See https://github.com/giantswarm/giantswarm/issues/1872